docs/vm/one-time-boot-for-iso-installation.md

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_position: 9
+sidebar_label: One-Time Boot For ISO Installation
+title: ""
+keywords:
+  - Harvester
+  - harvester
+  - One-Time
+  - one-time
+  - Boot
+  - boot
+  - ISO
+  - iso
+  - Installation
+  - installation
+  - ISO Installation
+  - iso installation
+  - One-Time Boot For ISO Installation
+Description: After installing the operating system using the one-time boot cd-rom, you can boot the operating system without unmounting the cd-rom.
+---
+
+# One-time Boot For ISO Installation
+
+When creating a VM to boot from cd-rom, you can use the **bootOrder** option so that the OS can boot from cd-rom during image installation, and boot from the disk when the installation is complete without unmounting the cd-rom.
+
+The following demo describes how to install an Ubuntu 22.04 Desktop ISO image:
+
+1. Click **Images** in the left sidebar and download the Ubuntu 22.04 Desktop ISO image.
+2. Click **Virtual Machines** in the leftside bar to create a VM. You need to set the VM basic configuration.
+3. Click the **Volumes** tab and set **Type** to `cd-rom`. In the **Image** field, select the image downloaded in step 1.
```

Comment:
the image type can be configured via the labels if it is not auto-detected, and this will help to save a lot of time for each VM creation.
<img width="2292" alt="image" src="https://user-images.githubusercontent.com/4569037/194750592-4b2c221b-f7da-4c28-bdea-5c91b3fa8676.png">

Reply:
in harvester-rc2, why I cant select any value for the labels. Can I use edit yaml instead.